src/main/java/org/team1540/robot2018/Robot.java

```diff
@@ -163,6 +118,7 @@ public void disabledInit() {
 
   @Override
   public void autonomousInit() {
+    // TODO: Move auto logic into command
```

Comment:
It should actually be here, the command framework does not like conditionals

Reply:
Maybe not a command but at it should at least go in a separate class/function.

Reply:
If you want to divide it into a function in `Robot` be my guest (you'd run into some issues with instance fields otherwise)